app/views/shared/_navbar.html.erb

```diff
@@ -30,6 +30,13 @@
         <li class="nav-item active">
           <%= link_to "Events", events_path, class: "nav-link" %>
         </li>
+
+
+      <%# # Make this #show instead of #index  %>
+        <li class="nav-item active">
+          <%= link_to "Organization", organizations_path, class: "nav-link" %>
```

Comment:
If i understand well - i think this link should lead to organization_path(current_cuser.organization.id) -- or smthg like that

Users should actually never be able to see a Organization#index page (I know we thought differently at the beginning)

Reply:
Yes you are right, I was trying to make it show_organization_path but that did not work, I will check with the TAs how to make it #show instead of #index. Left it as is just to show them an example.